docker/ctest_sirf.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
+# source env 
+export INSTALL_DIR=~/install
+source ${INSTALL_DIR}/bin/env_sirf.sh
+
+# append the path of virtualenv
+export PATH=/home/runner/.local/bin:$PATH
+source ~/devel/install/bin/activate 
+
+${INSTALL_DIR}/bin/gadgetron >& gadgetron.log&
```

Comment:
```suggestion
gadgetron >& gadgetron.log&
```
as it's already in the path